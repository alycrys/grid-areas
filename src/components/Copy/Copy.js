import React from "react";
import PropTypes from "prop-types";
import styles from "./Copy.module.css";

const Copy = ({ title, content }) => {
  return (
    <div className={styles.copy}>
      <div className={styles.title}>{title}</div>
      <div className={styles.content}>{content}</div>
    </div>
  );
};

Copy.propTypes = {
  title: PropTypes.string.isRequired,
  content: PropTypes.string.isRequired,
};

export default Copy;
