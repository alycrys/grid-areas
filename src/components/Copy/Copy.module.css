
.title {
  font-size: 18px;
  text-align: left;
}

.content {
  font-size: 12px;
  text-align: left;
  max-height: 45px;
  overflow: hidden;
}
