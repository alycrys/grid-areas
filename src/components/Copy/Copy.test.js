import React from "react";
import renderer from "react-test-renderer";
import Copy from "./Copy";

it("renders correctly", () => {
  const wrapper = renderer.create(<Copy title="title" content="content" />);

  expect(wrapper.toJSON()).toMatchSnapshot();
});
