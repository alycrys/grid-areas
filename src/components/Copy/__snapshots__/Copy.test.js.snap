// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="copy"
>
  <div
    className="title"
  >
    title
  </div>
  <div
    className="content"
  >
    content
  </div>
</div>
`;
